### ISSUE TITLE
The idea of presenting several models side by side in one window

### ISSUE THREAD

**Issue by Lukasz858585** [upscayl/upscayl#173]
Would it be possible to preview several models in one window, for example 4 or 2 next to each other. What I mean is that first we select the models we want to preview, and then after processing the program will create four separate windows presenting the effect of each model ?

**Comment by aaronliu0130** [comment]
Won't previewing require you to process it first? That would take a lot of time and energy

**Comment by NayamAmarshe** [comment]
Yeah, no plans for such a thing yet. A better option would be to just upscale the images and then open them side by side.

**Comment by aaronliu0130** [comment]
For those who want this feature: would #1283 solve your problem?
